lang/cs/Org.Apache.REEF.Client/Avro/YARN/AvroYarnJobSubmissionParameters.cs

```diff
@@ -60,6 +60,12 @@ public static string Schema
         public string jobSubmissionDirectoryPrefix { get; set; }
 
         /// <summary>
+        /// Gets or sets file system url
```

Comment:
So, are we using this as the `fs.defaultFS` parameter for Hadoop? If so, maybe the documentation should say so?